conf/map/battle/feature.conf

```diff
@@ -98,4 +98,8 @@ features: {
 	// true:  enable
 	// false: disable (default)
 	replace_refine_npcs: false
+
+	// Enable Multi-Critical damage
+	// Requires: 2016-12-07 RagexeRE or later
+	enable_multi_critical: true
```

Comment:
please set this to false otherwise the CI that checks different client types would fail